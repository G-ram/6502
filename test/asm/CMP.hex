a9 08 a2 09 8e 00 01 cd 00 01 
