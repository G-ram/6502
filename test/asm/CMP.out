A: 08
X: 09
Y: 00
P: 10010000
S: 00
PC: 60a
100 : 09
101 : 00
